src/plugin.js

```diff
@@ -226,6 +228,29 @@ const onPlayerReady = (player, emeError) => {
       setupSessions(player);
       handleWebKitNeedKeyEvent(event, getOptions(player), player.tech_)
         .catch(emeError);
+    };
+
+    // Support Safari EME with FairPlay
+    // (also used in early Chrome or Chrome with EME disabled flag)
+    player.tech_.el_.addEventListener('webkitneedkey', (event) => {
+      // It's possible that at the start of playback a rendition switch
+      // on a small player in safari's HLS implementation will cause
+      // two webkitneedkey events to occur. We want to make sure to cancel
+      // our first existing request if we get another within 1 second. This
+      // prevents a non-fatal player error from showing up due to a
+      // request failure.
+      if (!player.eme.webkitneedkey_.seenFirstEvent) {
+        player.clearTimeout(player.eme.webkitneedkey_.timeout);
```

Comment:
```suggestion
        // clear the old timeout so that a new one can be created with the new rendition's event data
        player.clearTimeout(player.eme.webkitneedkey_.timeout);
```